src/api/ApiResponder.php

```diff
@@ -232,11 +369,25 @@ protected function get_interface_response_loadPlayerName() {
         }
     }
 
+    /**
+     * Interface redirect for loadPlayerInfo
+     *
+     * @param BMInterface $interface
+     * @param array $args
+     * @return NULL|array
+     */
     protected function get_interface_response_loadPlayerInfo($interface) {
         $result = $interface->player()->get_player_info($_SESSION['user_id']);
         return $result;
     }
 
+    /**
+     * Interface redirect for savePlayerInfo
+     *
+     * @param BMInterface $interface
+     * @param array $args
+     * @return NULL|FALSE|array
```

Comment:
Yeah, this can't be right.  I mean, i believe you that it's true, but it can't be desirable for there to be a difference between NULL and FALSE, can it?